third_party/jpeg/BUILD.bazel

```diff
@@ -524,12 +524,12 @@ JCONFIG_NOWIN_COMMON_SUBSTITUTIONS = {
     "#cmakedefine HAVE_STDDEF_H": "#define HAVE_STDDEF_H 1",
     "#cmakedefine HAVE_STDLIB_H": "#define HAVE_STDLIB_H 1",
     "#cmakedefine NEED_SYS_TYPES_H": "#define NEED_SYS_TYPES_H",
-    "#cmakedefine NEED_BSD_STRINGS": "",
+    "#cmakedefine NEED_BSD_STRINGS 1": "",
```

Comment:
Should the 1 be moved to the right of `:`, inside the other pair of quotes? Both here and below

Reply:
@mihaimaruseac In 2.0.0 version of libjpeg-turbo the cmakedefine is:
```
#cmakedefine NEED_BSD_STRINGS
```

So in the past the replacement bazel rule is would be the following to undef:
```
"#cmakedefine NEED_BSD_STRINGS": "",
```

And the following for define:
```
"#cmakedefine NEED_BSD_STRINGS": "#define NEED_BSD_STRINGS 1",
```

However, in 2.0.4 libjpeg-turbo changed the way the template was setup. It is now:
```
#cmakedefine NEED_BSD_STRINGS 1
```

So in order to undef I use the following:
```
"#cmakedefine NEED_BSD_STRINGS 1": "",
```

And I use the following for define (note the extra `1` is actually not touched:
```
"#cmakedefine NEED_BSD_STRINGS": "#define NEED_BSD_STRINGS",
```

Reply:
Thank you for detailing